v22.2/known-limitations.md

```diff
@@ -10,6 +10,84 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### DROP OWNED BY does not support drop functions
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
```

Comment:
Link?

Reply:
Docs not live yet. Will have to add when Rich's PR is merged. Sounds like he may be able to do that shortly after mine. https://github.com/cockroachdb/docs/pull/15715#discussion_r1037463214